pkg/generators/markers.go

```diff
@@ -231,119 +231,202 @@ func ParseCommentTags(t *types.Type, comments []string, prefix string) (CommentT
 	return commentTags, nil
 }
 
-// lintArrayIndices attempts to prevent:
-// 1. Skipped indices
-// 2. Non-consecutive indices
-func lintArrayIndices(markerComments []string, prefix string) error {
+var (
+	allowedKeyCharacterSet = `[:_a-zA-Z0-9\[\]\-]`
+	valueEmpty             = regexp.MustCompile(fmt.Sprintf(`^(%s*)$`, allowedKeyCharacterSet))
+	valueAssign            = regexp.MustCompile(fmt.Sprintf(`^(%s*)=(.*)$`, allowedKeyCharacterSet))
+	valueRawString         = regexp.MustCompile(fmt.Sprintf(`^(%s*)>(.*)$`, allowedKeyCharacterSet))
+)
+
+// extractCommentTags parses comments for lines of the form:
+//
+//	'marker' + "key=value"
+//
+//	or to specify truthy boolean keys:
+//
+//	'marker' + "key"
+//
+// Values are optional; "" is the default.  A tag can be specified more than
+// one time and all values are returned.  Returns a map with an entry for
+// for each key and a value.
+//
+// Similar to version from gengo, but this version support only allows one
```

Comment:
Previous paragraph says "A tag can be specified more than one time and all values are returned." but this says "only allows one value".

Am I misunderstanding?

Reply:
Gah, that was missed when modifying the original comment. 

The correct doc should read only one value per key. I'll correct that